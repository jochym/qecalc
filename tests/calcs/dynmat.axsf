ANIMSTEPS   9
CRYSTAL
PRIMVEC
    3.175063252    0.000000000    0.000000000
   -1.587531626    2.749685434    0.000000000
    0.000000000    0.000000000    3.478437053
PRIMCOORD  1
      3   1
   Mg     0.00000   0.00000   0.00000   0.00747   0.05223  -0.02346
   B      1.58753   0.91656   1.73922   0.00747   0.05223  -0.02346
   B      0.00000   1.83312   1.73922   0.00747   0.05223  -0.02346
PRIMCOORD  2
      3   1
   Mg     0.00000   0.00000   0.00000   0.00402   0.02312   0.05275
   B      1.58753   0.91656   1.73922   0.00402   0.02312   0.05275
   B      0.00000   1.83312   1.73922   0.00402   0.02312   0.05275
PRIMCOORD  3
      3   1
   Mg     0.00000   0.00000   0.00000   0.05711  -0.00845  -0.00065
   B      1.58753   0.91656   1.73922   0.05711  -0.00845  -0.00065
   B      0.00000   1.83312   1.73922   0.05711  -0.00845  -0.00065
PRIMCOORD  4
      3   1
   Mg     0.00000   0.00000   0.00000  -0.00789   0.05333   0.00000
   B      1.58753   0.91656   1.73922   0.00872  -0.05891   0.00000
   B      0.00000   1.83312   1.73922   0.00872  -0.05891   0.00000
PRIMCOORD  5
      3   1
   Mg     0.00000   0.00000   0.00000  -0.05333  -0.00789   0.00000
   B      1.58753   0.91656   1.73922   0.05891   0.00872   0.00000
   B      0.00000   1.83312   1.73922   0.05891   0.00872   0.00000
PRIMCOORD  6
      3   1
   Mg     0.00000   0.00000   0.00000   0.00000   0.00000  -0.05391
   B      1.58753   0.91656   1.73922   0.00000   0.00000   0.05956
   B      0.00000   1.83312   1.73922   0.00000   0.00000   0.05956
PRIMCOORD  7
      3   1
   Mg     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   B      1.58753   0.91656   1.73922   0.06941  -0.01351   0.00000
   B      0.00000   1.83312   1.73922  -0.06941   0.01351   0.00000
PRIMCOORD  8
      3   1
   Mg     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   B      1.58753   0.91656   1.73922  -0.01351  -0.06941   0.00000
   B      0.00000   1.83312   1.73922   0.01351   0.06941   0.00000
PRIMCOORD  9
      3   1
   Mg     0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   B      1.58753   0.91656   1.73922   0.00000   0.00000  -0.07071
   B      0.00000   1.83312   1.73922   0.00000   0.00000   0.07071
