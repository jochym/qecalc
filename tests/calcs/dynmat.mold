[Molden Format]
[FREQ]
    0.00
    0.00
    0.00
  314.23
  314.23
  370.17
  499.00
  499.00
  660.83
[FR-COORD]
   Mg          0.00000        0.00000        0.00000
   B           3.00000        1.73205        3.28665
   B           0.00000        3.46410        3.28665
[FR-NORM-COORD]
 vibration     1
   0.07466   0.52225  -0.23455
   0.07466   0.52225  -0.23455
   0.07466   0.52225  -0.23455
 vibration     2
   0.04021   0.23117   0.52752
   0.04021   0.23117   0.52752
   0.04021   0.23117   0.52752
 vibration     3
   0.57109  -0.08455  -0.00648
   0.57109  -0.08455  -0.00648
   0.57109  -0.08455  -0.00648
 vibration     4
  -0.07894   0.53327   0.00000
   0.08721  -0.58914   0.00000
   0.08721  -0.58914   0.00000
 vibration     5
  -0.53327  -0.07894   0.00000
   0.58914   0.08721   0.00000
   0.58914   0.08721   0.00000
 vibration     6
   0.00000   0.00000  -0.53908
   0.00000   0.00000   0.59556
   0.00000   0.00000   0.59556
 vibration     7
   0.00000   0.00000   0.00000
   0.69409  -0.13506   0.00000
  -0.69409   0.13506   0.00000
 vibration     8
   0.00000   0.00000   0.00000
  -0.13506  -0.69409   0.00000
   0.13506   0.69409   0.00000
 vibration     9
   0.00000   0.00000   0.00000
   0.00000   0.00000  -0.70711
   0.00000   0.00000   0.70711
