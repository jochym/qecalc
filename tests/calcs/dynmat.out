     diagonalizing the dynamical matrix ...

 q =       0.0000      0.0000      0.0000
 **************************************************************************
     omega( 1) =       0.000000 [THz] =      -0.000005 [cm-1]
 (  0.074655   0.000000     0.522250   0.000000    -0.234552   0.000000   )
 (  0.074655   0.000000     0.522250   0.000000    -0.234552   0.000000   )
 (  0.074655   0.000000     0.522250   0.000000    -0.234552   0.000000   )
     omega( 2) =       0.000000 [THz] =      -0.000003 [cm-1]
 (  0.040208   0.000000     0.231171   0.000000     0.527519   0.000000   )
 (  0.040208   0.000000     0.231171   0.000000     0.527519   0.000000   )
 (  0.040208   0.000000     0.231171   0.000000     0.527519   0.000000   )
     omega( 3) =       0.000000 [THz] =       0.000005 [cm-1]
 (  0.571090   0.000000    -0.084546   0.000000    -0.006479   0.000000   )
 (  0.571090   0.000000    -0.084546   0.000000    -0.006479   0.000000   )
 (  0.571090   0.000000    -0.084546   0.000000    -0.006479   0.000000   )
     omega( 4) =       9.420167 [THz] =     314.225048 [cm-1]
 ( -0.078941   0.000000     0.533271   0.000000     0.000000   0.000000   )
 (  0.087212   0.000000    -0.589143   0.000000     0.000000   0.000000   )
 (  0.087212   0.000000    -0.589143   0.000000     0.000000   0.000000   )
     omega( 5) =       9.420167 [THz] =     314.225048 [cm-1]
 ( -0.533271   0.000000    -0.078941   0.000000     0.000000   0.000000   )
 (  0.589143   0.000000     0.087212   0.000000     0.000000   0.000000   )
 (  0.589143   0.000000     0.087212   0.000000     0.000000   0.000000   )
     omega( 6) =      11.097332 [THz] =     370.169606 [cm-1]
 (  0.000000   0.000000     0.000000   0.000000    -0.539082   0.000000   )
 (  0.000000   0.000000     0.000000   0.000000     0.595563   0.000000   )
 (  0.000000   0.000000     0.000000   0.000000     0.595563   0.000000   )
     omega( 7) =      14.959576 [THz] =     499.001049 [cm-1]
 (  0.000000   0.000000     0.000000   0.000000     0.000000   0.000000   )
 (  0.694089   0.000000    -0.135055   0.000000     0.000000   0.000000   )
 ( -0.694089   0.000000     0.135055   0.000000     0.000000   0.000000   )
     omega( 8) =      14.959576 [THz] =     499.001049 [cm-1]
 (  0.000000   0.000000     0.000000   0.000000     0.000000   0.000000   )
 ( -0.135055   0.000000    -0.694089   0.000000     0.000000   0.000000   )
 (  0.135055   0.000000     0.694089   0.000000     0.000000   0.000000   )
     omega( 9) =      19.810926 [THz] =     660.825750 [cm-1]
 (  0.000000   0.000000     0.000000   0.000000     0.000000   0.000000   )
 (  0.000000   0.000000     0.000000   0.000000    -0.707107   0.000000   )
 (  0.000000   0.000000     0.000000   0.000000     0.707107   0.000000   )
 **************************************************************************
